x-pack/test/functional/es_archives/security_solution/entity_store/data.json

```diff
@@ -17,16 +18,10 @@
         "hash": []
       },
       "entity": {
-        "last_seen_timestamp": "2024-09-11T11:24:15.588Z",
-        "schema_version": "v1",
```

Comment:
Do we no longer need these other fields?

Reply:
There is a "new" ingest pipeline step that deletes them from the index. 
Here is the code: https://github.com/elastic/kibana/blob/0617ad44406daecd0342a8fbaf84d9cdef8c5d50/x-pack/plugins/security_solution/server/lib/entity_analytics/entity_store/elasticsearch_assets/ingest_processor_steps/remove_entity_definition_fields_step.ts#L18